README.md

```diff
@@ -35,30 +52,44 @@ If you are operating Fn, this section is for you.
 * [Log Stores](fn/operate/log-stores.md)
 * [Message Queues](fn/operate/message-queues.md)
 * [Metrics](fn/operate/metrics.md)
-* [Operating on Windows](fn/operate/windows.md)
-* [Private Registries](fn/operate/private_registries.md)
-* [Runtime Options](fn/operate/options.md)
 * [User Interface](fn/operate/ui.md)
 
-### Advanced
+### Learn More
 
 * [Docker Configuration](fn/operate/docker.md)
 * [Extending Fn](fn/operate/extending.md)
 * [Kubernetes Helm Chart for Fn](https://github.com/fnproject/fn-helm/)
+* [Operating on Windows](fn/operate/windows.md)
+* [Private Registries](fn/operate/private_registries.md)
+* [Runtime Options](fn/operate/options.md)
 
 ## For Contributors
 
 If you are working on the Fn Project, want to work on it or are creating extensions, this section is for you.
 
 * [Writing Extensions](fn/develop/extensions.md)
 * [Using Annotations](fn/develop/annotations.md)
+* Join our [Slack Community](http://slack.fnproject.io)
+* See our new [Community Page](https://github.com/fnproject/docs/blob/master/COMMUNITY.md)
+* Learn how to [contribute](CONTRIBUTING.md)
+* Find [issues](https://github.com/fnproject/fn/issues) and become a contributor
+* Join us at one of our [Fn Events](http://events.fnproject.io) or even speak at one!
+* Coming in Q1'19: Regularly scheduled planning meetings for contributing to the Fn Project
+
+
+## Other Resources
 
-## [CLI Documentation](cli/README.md)
-* [CLI Reference](cli/README.md#fn-command-reference)
-* [CLI How-to Guides](cli/README.md#fn-cli-how-to-guides)
+* [Ask your question on StackOverflow](https://stackoverflow.com/questions/tagged/fn) and tag it with `fn`
+* Join our [Slack Community](http://slack.fnproject.io)
+* Join us at one of our [Fn Events](http://events.fnproject.io) or even speak at one!
+* Coming in Q1'19: Regularly scheduled planning meetings for contributing to the Fn Project
+* For a full presentation with lots of content you can use in your own presentations, see [The Fn Project Presentation Master](http://deck.fnproject.io)
+* Check out our sub-projects: [Flow](https://github.com/fnproject/flow), [UI](https://github.com/fnproject/ui), [FnLB](https://github.com/fnproject/lb)
 
-## [FDKs](fdks/README.md)
 
-## [Examples](examples/README.md)
+### Stay Informed
 
-## [Tutorials](https://fnproject.io/tutorials)
+* [Blog](https://medium.com/fnproject)
+* [Twitter](https://twitter.com/fnproject)
+* [YouTube](https://www.youtube.com/channel/UCo3fJqEGRx9PW_ODXk3b1nw)
```

Comment:
As David mentioned before, if we're not posting videos, then what's the point for promoting the channel?

Reply:
let's post some videos then ;)